uniffi_bindgen/src/bindings/swift/gen_swift/mod.rs

```diff
@@ -754,9 +760,13 @@ pub mod filters {
         Ok(quote_general_keyword(oracle().enum_variant_name(nm)))
     }
 
-    /// Like enum_variant_swift_quoted, but a class name.
-    pub fn error_variant_swift_quoted(nm: &str) -> Result<String, rinja::Error> {
-        Ok(quote_general_keyword(oracle().class_name(nm)))
+    /// Get the Swift rendering of an individual enum variant for an Error type.
```

Comment:
probably worth commenting here that the default name isn't actually preferred - ie, explaining why this exists?